quodlibet/quodlibet/qltk/views.py

```diff
@@ -257,18 +256,12 @@ def __motion(self, view, event):
         # the renderer xpad is not enough for the tooltip, especially with
         # rounded corners the label gets nearly clipped.
         MIN_HINT_X_PAD = 4
```

Comment:
if you change MIN_HINT_X_PAD to 20 for example the tooltips are missaligned with this change

Reply:
You're right— I didn't realize `extra_xpad` was already accounted for elsewhere. How does it look now?